src/packages/user/manifests.ts

```diff
@@ -17,4 +17,10 @@ export const manifests = [
 	...currentUserManifests,
 	...userPermissionManifests,
 	...modalManifests,
+	{
+		name: 'User Management Entry Point',
+		alias: 'Umb.EntryPoint.UserManagement',
+		type: 'entryPoint',
```

Comment:
Could we use a bundle instead?  Maybe it's just me but I find the Bundle easier to decode(read), as I then know it's not doing any custom code. It's just registering some more manifests.

Reply:
Actually, Just expends a manifest.js here, like done with the above manifests.

Reply:
I have registered it as a globalContext. Then the entry point is gone :)